src/Driver/SolverTypes/MISSolverType.jl

```diff
@@ -34,14 +34,23 @@ fast and slow dynamics respectively, depending on the state `Q`.
     Default: `MIS2`
 - `fast_method` (Function): Function defining the fast solver.
     Default: `LSRK54CarpenterKennedy`
-- `nsubsteps` (Int): Integer denoting the total number of times
+- `nsubsteps` (Tuple): Tuple denoting the total number of times
     to substep the fast process.
-    Default: `50`
+    Default: `(50,)`
 - `discrete_splitting` (Boolean): Boolean denoting whether a PDE level or
     discretized level splitting should be used. If `true` then the PDE is
     discretized in such a way that `f_fast + f_slow` is equivalent to
     discretizing the original PDE directly.
     Default: `false`
+- `hivi_splitting` (Boolean): Boolean denoting whether the fast processes need to
+    be split into their horizontal and vertical components. For instance this is
+    necessary, when a MIS method is chosen as the fast solver.
+    Default: `false`
+- `numerical_flux_first_order_fast` (NumericalFluxFirstOrder): The type of
```

Comment:
We should try to avoid passing these numerical flux arguments --- much of this could be inferred from the DG model that this solver type has access to.